components/nexus-security/pom.xml

```diff
@@ -45,23 +45,17 @@
     </dependency>
 
     <dependency>
-      <groupId>javax.enterprise</groupId>
-      <artifactId>cdi-api</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.eclipse.sisu</groupId>
-      <artifactId>org.eclipse.sisu.inject</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.sisu</groupId>
-      <artifactId>sisu-guice</artifactId>
+      <groupId>org.sonatype.sisu.goodies</groupId>
+      <artifactId>goodies-eventbus</artifactId>
     </dependency>
 
+    <!--
+    NOTE: Here only for javax.enterprise.inject.Typed, consider removing use and dropping dependency?
```

Comment:
You only need the `@Typed` hint if you want to restrict visibility of a component to a specific super-type. Looking at the various components I don't see any obvious need for them to be `@Typed`, so you could drop the annotations and remove this dependency.